src/core/ext/filters/client_channel/lb_policy/round_robin/round_robin.c

```diff
@@ -74,6 +74,9 @@ typedef struct round_robin_lb_policy {
   bool started_picking;
   /** are we shutting down? */
   bool shutdown;
+  /** has the policy gotten into the GRPC_CHANNEL_SHUTDOWN? No picks can be
+   * service after this point, the policy will never transition out. */
+  bool in_connectivity_shutdown;
```

Comment:
Could we use the existing `shutdown` bool for this, rather than adding a new one?  Not sure whether or not this makes sense; it just seems a little odd to have two different variables for very similar purposes.